backend/src/main/java/nl/itvitae/twitbook/user/UserDTO.java

```diff
@@ -2,8 +2,12 @@
 
 import java.time.LocalDateTime;
 
-public record UserDTO(String username, LocalDateTime registerDate) {
+public record UserDTO(String username, LocalDateTime registerDate, boolean hasFollowed, int followers, int following) {
```

Comment:
Maybe `numFollowers` and `numFollowing`? `following` and `hasFollowed` sound similar